libs/verification-common/src/blueprint_contracts.rs

```diff
@@ -0,0 +1,101 @@
+use bytes::Bytes;
+
+// ERC5202_PREFIX = b"\xFE\x71\x00"  # default prefix from ERC-5202
+// def blueprint_bytecode(self) -> bytes:
+//     blueprint_bytecode = ERC5202_PREFIX + self.bytecode
+//
+//     # the length of the deployed code in bytes
+//     len_bytes = len(blueprint_bytecode).to_bytes(2, "big")
+//     deploy_bytecode = b"\x61" + len_bytes + b"\x3d\x81\x60\x0a\x3d\x39\xf3"
+//
+//     return deploy_bytecode + blueprint_bytecode
+
+pub fn from_runtime_code(code: Bytes) -> Option<Bytes> {
+    let prefix = [0xfe, 0x71, 0x00];
```

Comment:
are you sure you need 0x00 at the end?

from eip: https://eips.ethereum.org/EIPS/eip-5202#reference-implementation

<img width="453" alt="image" src="https://github.com/blockscout/blockscout-rs/assets/41516657/dbdfec23-9466-439a-a18d-7331c36d9707">

Reply:
Took it from vyper implementation for now, which does not have any data supported and, thus, always have the third byte as 0x00

Probably you are right and it make sense to implement it more generically according to ERC standard

Reply:
Fixed